src/main/java/com/example/demo/controller/ItemController.java

```diff
@@ -76,39 +105,99 @@ public String newItem(Model model, @ModelAttribute Item item) {
 	}
 
 	@PostMapping("/new")
-	public String create(@Validated @ModelAttribute Item item, BindingResult result) {
+	public String create(Model model, @Validated @ModelAttribute Item item, BindingResult result) {
+
 		if (result.hasErrors()) {
+			model.addAttribute("title", "商品APP_新規登録");
+			model.addAttribute("logo", "AyaDesign");
+			Map<String, String> radioCategory = itemService.initRadioCategory();
+			model.addAttribute("radioCategory", radioCategory);
 			return "items/new";
 		}
 		itemService.insertOne(item);
 		return "redirect:/items/index";
 	}
 
 	/* 更新 */
-	@GetMapping("edit/id={id}")
+	@GetMapping("/edit/id={id}")
 	public String edit(@PathVariable("id") int id, Model model) {
 
 		model.addAttribute("title", "商品APP_更新");
 		model.addAttribute("logo", "AyaDesign");
 
-		model.addAttribute("item", itemService.findById(id));
+		model.addAttribute("item", itemService.findById(id).get());
 		Map<String, String> radioCategory;
 		radioCategory = itemService.initRadioCategory();
 		model.addAttribute("radioCategory", radioCategory);
 		return "items/edit";
 	}
 
-	@PostMapping("edit/id={id}")
-	public String update(@ModelAttribute Item item, Model model) {
+	@PostMapping("/edit/id={id}")
```

Comment:
formのidつかっているので↓でよい？
```suggestion
	@PostMapping("/edit")
```